### PR TITLE
fix: add BEE_FULL_NODE into docker-compose file

### PR DESCRIPTION
**Pull request by rootial**
this is necessary for docker to start in full node

---
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1840)

### CODE REVIEW

**Comment by Eknir** [comment]
Hi there,
The default behavior of our node and packages is to start in light node mode. If you wish, it would be great to add this explicitly in the README.md.

**Comment by rootial** [comment]
@Eknir  just add full node instruction in the readme. please take a look. thanks

File: packaging/docker/docker-compose.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ services:
       - BEE_DB_DISABLE_SEEKS_COMPACTION
       - BEE_DEBUG_API_ADDR
       - BEE_DEBUG_API_ENABLE
+      - BEE_FULL_NODE
```
Comment (Eknir) [inline packaging/docker/docker-compose.yml:32]: Please remove this line, then your PR is good to go!

Comment (rootial) [reply packaging/docker/docker-compose.yml:32]: this line is necessary if we need to run node in full mode by setting `BEE_FULL_NODE` in env file.

**Review by Eknir — APPROVED** [review]
LGTM! Thanks

**Comment by rootial** [comment]
@vandot  can u take a look this pr?

File: packaging/docker/docker-compose.yml (hunk 1 above)
Comment (Eknir) [reply packaging/docker/docker-compose.yml:32]: You are right!

**Review by vandot — APPROVED** [review]

**Pull request merged**